sprint1.2.css

```
@@ -0,0 +1,481 @@
+    padding-right: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: space-evenly;
+}
+.s3-hotitem
+{
+    width: 100%;
+    height: 1.625rem;
+    font-weight: 700; 
+    font-size: 1.125rem;
+    line-height: 1.625rem;
+    color: #3692ff;
+}
+.s3-best
+{
+    width: 20.9375rem;
+    height: 12.5rem;
+    gap: 1.5rem;      
+    display: flex;
+    flex-direction: column;
+}
+.s3-bestitem
+{
+    width: 18.3125rem;
+    height: 7rem;
+    font-weight: 700;
+    font-size: 2.38rem;
+    line-height: 3.5rem;
+    color: #374151;
+
+}
+.s3-bestcheck
+{
+    width: 20.9375rem;
+    height: 4rem;
+    font-weight: 500;
+    font-size: 1.4rem;
+    line-height: 2rem;
+    color: #374151;
+}
+/* 아래 배너 */
+.final-shot
+{
+    width: 120rem;
+    height: 33.75rem;
+    display: flex;
+    align-items: flex-end;
+    justify-content: center;
+    gap: 4.3125rem;
+}
+.fi-mentbox
+{   
+    width: 18.4375rem;
+    height: 24.8125rem;
+    gap: 0.625rem;    
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+}
+.fi-ment
+{
+    width: 18.4375rem;
+    height: 10.75rem;
+    font-weight: 700;
+    font-size: 2.3rem;
+    line-height: 3.5rem;
+    color: #374151;
+}
+.fi-hipanda
+{
+    width: 46.625rem;
+    height: 24.8125rem;
+    
+}
+/* 맨 아래 */
+footer
+{
+    width: 100%;
+    height: 11.25rem;
+    margin: 0 auto;
+    padding-top: 2rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    align-content: flex-start;
+    gap: 0.625rem;
+    background-color: #111827;
+}
+.foot-line
+{
+    gap: 336.5px;
+    width: 100rem;
+    height: 1.25rem;
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+}
+.foot-box
+{
+    width: 7rem;
+    height: 1.1875rem;
+}
+.foot-ment
+{
+    color: #9CA3AF;
+    font-weight: 400;
+    font-size: 1rem;
+    line-height: 1.193125rem;
+    text-align: left;
+}
+.foot-box2
+{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 9.9375rem;
+    height: 1.1875rem;
+    gap: 1.875rem;    
+}
+.foot-pri
+{
+    width: 6.25rem;
+    height: 1.1875rem;
+    font-weight: 400;
+    font-size: 0.9rem;
+    line-height: 1.193125rem;
+    text-align: left;
+    text-decoration: none;
+    color: #e5e7eb;
+}
+.foot-faq
+{
+    width: 1.8125rem;
+    height: 1.1875rem;
+    font-weight: 400;
+    font-size: 1rem;
+    line-height: 1.193125rem;
+    text-align: right;
+    text-decoration: none;
+    color: #E5E7EB;
+}
+.foot-pri.foot-faq {
+    text-decoration: none;
+}
+.foot-box3
+{
+    display: flex;
+    justify-content: space-between;
+    width: 7.25rem;
+    height: 1.25rem;
+    gap: 0.75rem;
+}
+.foot-face
+{
+    width: 1.25rem;
+    height: 1.25rem;
+} 
+.foot-twi  
+{
+    width: 1.25rem;
+    height: 1.25rem;
+}
+.foot-ytub
+{
+    width: 1.25rem;
+    height: 1.25rem;
+}
+.foot-insta
+{
+    width: 1.25rem;
+    height: 1.25rem;
+}
```

💊 제안
css가 반복사용되므로 하나의 클래스만 만들면 사용하시면 더 좋을 것 같습니다.

```suggestion
.footer-item {
    width: 1.25rem;
    height: 1.25rem;
}
```